pyout/tabular.py

```diff
@@ -526,6 +528,13 @@ def __call__(self, row, style=None):
         row = self._normalizer(row)
         callables = self._strip_callables(row)
 
+        # Fill in any missing values.
+        for column in self._columns:
+            if column in row:
+                continue
+            missing = self._style[column].get("missing")
+            row[column] = Nothing(missing) if missing else NOTHING
```

Comment:
so `missing` has to be a string? or otherwise what would happen if I specify `missing=0`?

Reply:
I guess I should just try, but what is happening to the Nothing values -- do they get styled as well? (just trying to grasp how could I (ab)use specification of `missing` ;) )

Reply:
The schema currently specifies the missing has to be a string, so the
validation should fail if missing=0.  We could of course lift that
constraint and then check `is None` here.

Re: styling: Should go through the normal styling.  It's just the
transformers that won't get called for a Nothing type.